src/main/kotlin/bsu/cc/parser/XlsxParser.kt

```diff
@@ -0,0 +1,152 @@
+package bsu.cc.parser
+
+import org.apache.poi.ss.usermodel.Cell
+import org.apache.poi.ss.usermodel.CellType
+import org.apache.poi.ss.usermodel.Row
+import org.apache.poi.ss.usermodel.Sheet
+import org.apache.poi.xssf.usermodel.XSSFWorkbook
+import java.io.File
+import java.io.FileInputStream
+import java.lang.IllegalArgumentException
+import java.lang.IllegalStateException
+import java.lang.reflect.Type
+import java.util.*
+
+
+
+fun readWorkbook(fileName: String): XSSFWorkbook {
+    val file = FileInputStream(File(fileName))
+    return XSSFWorkbook(file)
```

Comment:
Was there a particular reason for using a variable? A function like this would probably be written more idiomatically as:
`fun readWorkbook(fileName: String): XSSFWorkbook = XSSFWorkbook(FileInputStream(File(fileName)))`

Reply:
I try to keep one liners from getting too complex, and this used to have a couple more chained method calls, so I split it, just never switched it back.